### PR TITLE
Add Styling

### PR DESCRIPTION
**Pull request by moise-mulungu**
On this milestone, I:
- Used the Zeplin credentials for the project
- Styled the navbar, and the book
- Used the information detailed in Zeplin 
- Updated the README file
- Fixed linter errors

### CODE REVIEW

File: src/components/Navigation.js [hunk 1]
```diff
@@ -3,14 +3,17 @@ import { Link } from 'react-router-dom';
...
+    <nav className="navContainer">
+      <div className="navItems">
+        <h1 id="navTitle">Bookstore CMS</h1>
+        <ul className="navPages">
           <li><Link to="/">Books</Link></li>
           <li><Link to="/categories">Categories</Link></li>
         </ul>
       </div>
+      <div className="icon">
+        <img src="https://img.icons8.com/external-anggara-outline-color-anggara-putra/32/000000/external-avatar-interface-anggara-outline-color-anggara-putra.png" alt="profile" />
```
Comment (micheaol) [inline src/components/Navigation.js]: Your app is not performing as expected. It keeps refreshing every second. Please see the video attached for the behavior. Kindly fix it.

https://www.loom.com/share/7ac6af2d4150452f9a756f3f04c5995d

File: src/components/Book.js [hunk 2]
```diff
@@ -13,22 +13,37 @@ export default function Book({ book }) {
 
   return (
     <div className="bookCard">
-      <div className="">
+      <div className="bookDetails">
         <p className="genre">{book.category}</p>
         <p className="title">{book.title}</p>
         <p className="author">{book.author}</p>
-        <ul>
```
Comment (micheaol) [inline src/components/Book.js:20]: Please ensure that the styling is EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/)tyling is EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/).

Your Implementation | Design
:---:|:---:
image | image

Your Implementation | Design
:---:|:---:
image | image


Your Implementation | Design
:---:|:---:
image | image

File: src/App.css [hunk 3]
```diff
@@ -1,13 +1,47 @@
...
+  padding: 1% 5%;
+  background-color: #fff;
+  width: 90%;
+}
+
+#navTitle {
+  font-weight: bold;
+  color: #0290ff;
+  width: 15rem;
+  height: 2.313rem;
```
Comment (micheaol) [inline src/App.css:23]: Please ensure that you use the same `font attributes` as in the original design. To get the fonts or the color, click on the element you want to inspect and you will see the `right pan` like the image below:

image

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @moise-mulungu , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by DeliceLydia — APPROVED** [review]
## Hi @moise-mulungu, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**